go-packages/meep-model/validator.go

```diff
@@ -96,6 +98,7 @@ var Version150 = semver.Version{Major: 1, Minor: 5, Patch: 0}
 var Version151 = semver.Version{Major: 1, Minor: 5, Patch: 1}
 var Version153 = semver.Version{Major: 1, Minor: 5, Patch: 3}
 var Version168 = semver.Version{Major: 1, Minor: 6, Patch: 8}
+var Version182 = semver.Version{Major: 1, Minor: 8, Patch: 2}
 
```

Comment:
Current validator version must also be bumped to 1.8.2

Reply:
Addressed.